inc/comments-template.php

```diff
@@ -13,7 +13,8 @@
 	<div id="coral_talk_<?php echo absint( $rand ); ?>"></div>
 	<script src="<?php echo esc_url( $talk_url . '/embed.js' ); ?>" async onload="
 		Coral.Talk.render(document.getElementById('coral_talk_<?php echo absint( $rand ); ?>'), {
-			talk: '<?php echo esc_url( $talk_url ); ?>'
+			talk: '<?php echo esc_url( $talk_url ); ?>',
+			asset_id: '<?php echo esc_js( coral_talk_get_asset_id() ); ?>'
```

Comment:
VIP prefers `wp_json_encode` to `esc_js` these days. Anyway to make it compatible with that? If it's just a single value, removing the containing quotes would do the trick.

Another thought would be to create this as a PHP array and then `wp_json_encode` the whole thing.